src/test/scala/org/kibanaLoadTest/helpers/ESClient.scala

```diff
@@ -14,11 +14,13 @@ import org.elasticsearch.common.xcontent.XContentType
 import org.kibanaLoadTest.ESConfiguration
 import org.slf4j.{Logger, LoggerFactory}
 import io.circe.Json
+import org.elasticsearch.action.bulk.BulkRequest
+
 import java.io.IOException
-import scala.collection.parallel.CollectionConverters._
 
 class ESClient(config: ESConfiguration) {
   val logger: Logger = LoggerFactory.getLogger("ES_Client")
+  val BULK_SIZE = 200
```

Comment:
Would it be useful for this to be configurable, like an environment variable?

Reply:
Agree, I will allow to pass env variable